build.gradle.kts

```diff
@@ -27,6 +28,14 @@ tasks.register("installGitHooks", Exec::class.java) {
     }
 }
 
+detekt {
+    toolVersion = "1.23.1"
```

Comment:
Please use detekt version from `version catalog`, it will make easy for maintaining tool version.